EnglishFilter/sections/whitelist.txt

```diff
@@ -120,6 +120,8 @@ youtube.com#$##header.ytd-c4-tabbed-header-renderer { position: absolute !import
 inc42.com#@#.tag-social
 @@||piano.io^$domain=inc42.com
 @@||tinypass.com^$domain=inc42.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/101793
+@@||publisher.inmobi.com^$domain=inmobi.com

```

Comment:
I don't think that we should the exception rule -  it's caused by AdGuard DNS. @AdamWr what do you think?

Reply:
It's also blocked by `AdGuard Mobile Ads filter`, but as far as I understand, it's advertising company, so I'm not sure if it should be unblocked. By the way, their main website is also broken - `https://www.inmobi.com/`
I guess that it would be better if user would add this rule to "User rules".